.github/workflows/publish-prod-storybook.yml

```diff
@@ -31,6 +31,24 @@ jobs:
             - name: Build Storybook
               run: yarn build:storybook
 
+            - name: Create and Checkout 'gh-pages' branch
+              run: |
+                git checkout --orphan gh-pages
+                git rm -rf .
+                git reset -- 'PR-*'
```

Comment:
Removing all files from the branch and then restoring any directories that start with the `PR-` syntax - hoping to set the foundation for PR-specific storybook builds